src/definitions/globals.ts

```diff
@@ -0,0 +1,121 @@
+const CLASSES = [
```

Comment:
This would require a tool (or even something built into the extension) to extract the documentation information from the Crystal API docs then parse it into this format. Doing something like this would be too much work for little benefit ([see how Docr does it](https://github.com/devnote-dev/docr/blob/4a22e5b9baa55837443b0380bbe35fe1bd7ae14c/src/docr/search.cr)).

Reply:
That makes sense to me. We'll just have to do our best to stay on top of changes to the compiler itself.